v2/pkg/cli/executor.go

```diff
@@ -361,12 +361,6 @@ func (o *ExecutorSchema) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	if o.Opts.Mode == mirrorToDisk {
-		// ensure working dir exists
-		err := os.MkdirAll(workingDir, 0755)
-		if err != nil {
-			o.Log.Error(" %v ", err)
-			return err
-		}
```

Comment:
I thought it was a good idea to ensure that workingDir existed before starting. Is this being initialized elsewhere?

Reply:
I removed it as v2 does not use working dir - but just for sanity sake I added it back again :)